frontend/src/components/form/index.tsx

```diff
@@ -29,15 +33,16 @@ Form.Title = function FormTitle({ children, ...restProps }: any) {
   return <Title {...restProps}>{children}</Title>
 }
 
-Form.Input = function FormInput({ setInput, ...restProps }: any) {
+Form.Input = function FormInput({ handleInputChange, ...restProps }: any) {
```

Comment:
I preferer props to be called with the `on` prefix, i.e `onChange` instead of `handleChange`.

Also, no need to add the `Input` word in the middle since we're already in input domain